### DISCUSSION TITLE
GitHub Actions

### DISCUSSION THREAD

**Discussion by vkrngn** [community/community#66480] [Actions]
### Select Topic Area

Question

### Body

GitHub Actions are disabled for my account?

**Comment by favianaldo** [comment]
Are you joking?

If it's disabled, you just don't have to open the github

If you delete an account, click the profile icon, then go to account, scroll and click delete your account

**Comment by LeandroLuna** [comment]
Hello, Vkrngn!

Are you getting any kind of error when using it? I don't think you need to enable anything to use the actions..
